libfwupdplugin/fu-device.h

```diff
@@ -585,6 +585,16 @@ typedef guint64 FuDeviceInternalFlags;
  */
 #define FU_DEVICE_INTERNAL_FLAG_NO_GENERIC_GUIDS (1ull << 36)
 
+/**
+ * FU_DEVICE_INTERNAL_FLAG_ENFORCE_REQUIRES:
+ *
+ * The device uses a generic instance ID and firmware requires a parent, child, sibling or CHID
+ * requirement.
+ *
+ * Since: 1.9.8
+ */
+#define FU_DEVICE_INTERNAL_FLAG_ENFORCE_REQUIRES (1llu << 37)
```

Comment:
why did you swap this to `llu` instead of `ull`?  All the others have `ull`

Reply:
Ohh, it was originally in libfwupd which confusingly has the other one. **Choose one** and I'll convert the codebase in another PR. :)

Reply:
Hah. I don't care, just make it consistent ;)